public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -123,7 +123,7 @@ export class SharedPreferences extends PureComponent<Props, State> {
           <Select
             value={dashboards.find(dashboard => dashboard.id === homeDashboardId)}
             getOptionValue={i => i.id}
-            getOptionLabel={i => i.title}
+            getOptionLabel={i => (typeof(i.folderTitle) !== "undefined" && i.folderTitle !== '') ? i.folderTitle + ' / ' + i.title : i.title}
```

Comment:
can you extract this to a function instead of inline, will make it a bit more readable